docs/user-guide/workflow-setup.md

```diff
@@ -0,0 +1,188 @@
+# Workflow Setup
+
+The first step in creating a new workflow is to choose what type of crawl you want to run. Crawl types are fixed and cannot be converted or changed later.
+
+`URL List`{ .badge-blue }
+:   The crawler visits every URL specified in a list, and optionally every URL linked on those pages.
+
+`Seeded Crawl`{ .badge-orange }
+:   The crawler automatically discovers and archives pages starting from a single seed URL.
+
+## Scope
+
+### List of URLs
+
+`URL List`{ .badge-blue } `Seeded Crawl`{ .badge-orange }
+
+This list informs the crawler what pages it should capture as part of a URL List crawl.
+
+It is also available under the _Additional URLs_ section for Seeded Crawls where it can accept arbitrary URLs that will be crawled regardless of other scoping rules.
+
+### Include Any Linked Page
+
+`URL List`{ .badge-blue }
+
+When enabled, the crawler will visit all the links it finds within each page defined in the _List of URLs_ field.
+
+??? tip "Crawling tags & search queries with URL List crawls"
+    This setting can be useful for crawling the content of specific tags or searh queries. Specify the tag or search query URL(s) in the _List of URLs_ field, e.g: `https://example.com/search?q=tag`, and enable _Include Any Linked Page_ to crawl all the content present on that search query page.
+
+### Crawl Start URL
+
+`Seeded Crawl`{ .badge-orange }
+
+This is the first page that the crawler will visit. It's important to set _Crawl Start URL_ that accurately represents the scope of the pages you wish to crawl as the _Start URL Scope_ selection will depend on this field's contents.
+
+You must specify the protocol (likely `http://` or `https://`) as a part of the URL entered into this field.
+
+### Start URL Scope
+
+`Seeded Crawl`{ .badge-orange }
+
+`Hashtag Links Only`
+:   This scope will ignore links that lead to other addresses such as `example.com/path` and will instead instruct the crawler to visit hashtag links such as `example.com/#linkedsection`.
+
+    This scope can be useful for crawling certian web apps that may not use unique URLs for their pages.
```

Comment:
```suggestion
    This scope can be useful for crawling certain web apps that may not use unique URLs for their pages.
```

Reply:
Oops!  I will do another round of spell checking... Thought my plugin should be working for that but maybe not 👀